protocol/app/upgrades/v5.1.0/upgrade_container_test.go

```diff
@@ -0,0 +1,187 @@
+//go:build all || container_test
```

Comment:
Nothing in state to test after the upgrade, just an empty test to run the upgrade.